src/dtproject/dtproject.py

```diff
@@ -62,6 +62,7 @@ def as_dict(self) -> Dict[str, dict]:
         "options": LayerOptions,
         "containers": LayerContainers,
         "devcontainers": LayerDevContainers,
+        "configurations" : LayerConfigurations
     }
     KNOWN_LAYERS = {**REQUIRED_LAYERS, **OPTIONAL_LAYERS}
 
```

Comment:
Merge `Configurations` and `Containers`, they do the same thing. Prefer the name `Containers` over `Configurations`